src/ui/pages/CreateSSIAgent/CreateSSIAgent.tsx

```diff
@@ -0,0 +1,226 @@
+  };
+
+  const handleValidate = async () => {
+    setLoading(true);
+    try {
+      await Agent.agent.bootAndConnect({
+        bootUrl: ssiAgent.bootUrl || "",
+        url: ssiAgent.connectUrl || "",
+      });
+
+      const { nextPath, updateRedux } = getNextRoute(RoutePath.SSI_AGENT, {
+        store: { stateCache },
+      });
+
+      updateReduxState(
+        nextPath.pathname,
+        {
+          store: { stateCache },
+        },
+        dispatch,
+        updateRedux
+      );
+
+      ionRouter.push(nextPath.pathname, "forward", "push");
+      handleClearState();
+    } catch (e) {
+      const errorMessage = (e as Error).message;
+      if (Agent.KERIA_BOOTED_ALREADY_BUT_CANNOT_CONNECT === errorMessage) {
+        setHasMismatchError(true);
+      }
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  const scanBootUrl = (event: ReactMouseEvent<HTMLElement, MouseEvent>) => {
+    event.stopPropagation();
+    dispatch(setCurrentOperation(OperationType.SCAN_SSI_BOOT_URL));
+  };
+
+  const scanConnectUrl = (event: ReactMouseEvent<HTMLElement, MouseEvent>) => {
+    event.stopPropagation();
+    dispatch(setCurrentOperation(OperationType.SCAN_SSI_CONNECT_URL));
+  };
+
+  return (
+    <>
+      <ScrollablePageLayout
+        pageId={pageId}
+        header={
+          <PageHeader
+            backButton={true}
+            beforeBack={handleClearState}
+            currentPath={RoutePath.SSI_AGENT}
+            progressBar={true}
+            progressBarValue={1}
+            progressBarBuffer={1}
+          />
+        }
+      >
+        {loading && (
+          <div
+            className="ssi-spinner-container"
+            data-testid="ssi-spinner-container"
+          >
+            <IonSpinner name="circular" />
+          </div>
+        )}
+        <h2 data-testid={`${pageId}-title`}>{i18n.t("ssiagent.title")}</h2>
+        <p
+          className="page-paragraph"
+          data-testid={`${pageId}-top-paragraph`}
+        >
+          {i18n.t("ssiagent.description")}
+        </p>
+        <div>
+          <IonButton
+            fill="outline"
+            className="copy-button secondary-button"
+            onClick={() => setOpenInfo(true)}
+          >
+            <IonIcon
+              slot="start"
+              icon={informationCircleOutline}
+            />
+            {i18n.t("ssiagent.button.info")}
+          </IonButton>
+        </div>
+        <CustomInput
+          dataTestId="boot-url-input"
+          title={`${i18n.t("ssiagent.input.boot.label")}`}
+          placeholder={`${i18n.t("ssiagent.input.boot.placeholder")}`}
+          actionIcon={scanOutline}
+          action={scanBootUrl}
+          onChangeInput={(bootUrl: string) => dispatch(setBootUrl(bootUrl))}
+          value={ssiAgent.bootUrl || ""}
+          onChangeFocus={setTouchedBootUrlInput}
+          error={!!displayBootUrlError}
+        />
+        {!!displayBootUrlError && (
+          <ErrorMessage
```

Comment:
when this error appears, the next component is pushed down. We should have a fixed height so that this does not happen